react/features/base/config/functions.native.ts

```diff
@@ -42,3 +42,14 @@ export function _cleanupConfig(config: IConfig) {
 export function getReplaceParticipant(state: IReduxState): string {
     return getFeatureFlag(state, REPLACE_PARTICIPANT, false);
 }
+
+/* eslint-disable */
```

Comment:
Why are you disabling eslint here? If it's because of an unused variable then prefix it with _ and it should go away.

Reply:
thanks for the tip, fixed and pplied to `_cleanupConfig` as well.